docs/specs/alcproxies.md

```diff
@@ -171,10 +167,8 @@ Instead of moving a specific type through a `BinaryFormatter`, using `DataContra
 
 Benefits of `DataContractSerializer`:
 * It may be easier compared to `BinaryFormatter` to cast this effectively across the type-isolation barrier.
-* Adding attributes is similar to `BinaryFormatter`, so any type should be able to make it across the barrier.
-
-Concerns with `DataContractSerializer`:
-* Many similar concerns to `BinaryFormatter`, such as potential security issues and the manual use of [DataContractAttribute] and [DataMemberAttribute] needing to be used. There may also be limits to what types can be serialized using `DataContractSerializer`.
+* There's no need to add attributes if you want the default behavior for `DataContractSerializer` (Serializes all public members).
+* There are none of the security issues compared to `BinaryFormatter` (or at least there are fewer).
```

Comment:
There are no known security issues with DCS. Fewer is saying > 0 which isn't the case.

Reply:
How about "It doesn't have the security issues known to exist in `BinaryFormatter`"?